source/features/show-whitespace.css

```diff
@@ -16,3 +16,49 @@
 	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z" fill="rgba(128, 128, 128, 50%25)"/%3E%3C/svg%3E');
 	background-size: 1ch 1.25em;
 }
+
+[data-tab-size='1'] {
+	--tab-size: 1;
+}
+
+[data-tab-size='2'] {
+	--tab-size: 2;
+}
+
+[data-tab-size='3'] {
+	--tab-size: 3;
+}
+
+[data-tab-size='4'] {
+	--tab-size: 4;
+}
+
+[data-tab-size='5'] {
+	--tab-size: 5;
+}
+
+[data-tab-size='6'] {
+	--tab-size: 6;
+}
+
+/* File blocks in comments always have [data-tab-size=8] #4833 */
+[data-tab-size='8']:not(.highlight) {
+	--tab-size: 8;
```

Comment:
Oh I see what you meant. As long as this doesn't exclude other pieces in our Tests it looks great.

Reply:
`--tab-size` must always match `tab-size`. If `tab-size.css` wants to override this, it should take care of all of it, otherwise we risk losing `--tab-size` sync or selector sync

Reply:
> `--tab-size` must always match `tab-size`. If `tab-size.css` wants to override this, it should take care of all of it, otherwise we risk losing `--tab-size` sync or selector sync

That's one way to see it, and the reason why [I asked for comments above](https://github.com/refined-github/refined-github/pull/6480#discussion_r1161304645)

But the situation is they depend on each use unless `tab-size` doesn't set or use `var(--tab-size)`